openvdb_houdini/houdini/GEO_PrimVDB.cc

```diff
@@ -1928,6 +1928,30 @@ geoIntersect(GridTypeA& grid_a, const GridTypeB &grid_b)
     }
 }
 
+/// This class is used as a functor to set inactive voxels to the background
+/// value.
+template<typename GridType>
+class geoInactiveToBackground
+{
+public:
+    typedef typename GridType::ValueOffIter             Iterator;
+    typedef typename GridType::ValueType                ValueType;
+
+    geoInactiveToBackground(const GridType& grid)
+    {
+        background = grid.background();
+    }
+
+    inline void operator()(const Iterator& iter) const
+    {
+        iter.setValue(background);
+        iter.setValueOff();
```

Comment:
This line isn't needed given that you're iterating over inactive values only.  It's also not strictly thread-safe, although that's probably OK in this case since you're not actually changing the values of any active masks.

Reply:
Thank you!  Not sure how that slipped in there. There was some concern that setValue() was implicitly activating; but it isn't.